routers/posts.py

```
@@ -0,0 +1,79 @@
+    posts = read_json("posts.json",default=[])
+    pass
+
+# 게시글 상세 조회
+@router.get("/{post_id}",response_model=PostOut)
+def get_post(post_id: int, user=Depends(get_current_user)):
+    posts = read_json("posts.json",default=[])
+    pass
+
+# 게시글 작성
+@router.post("/",response_model=PostOut)
+def create_post(payload: PostCreate, user=Depends(get_current_user)):
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)